src/input.js

```diff
@@ -67,14 +67,16 @@ const renameKeys = curry((keysMap, obj) => reduce((acc, key) =>
 
 const objectToChoices = toPairs & map(([value, name]) => ({ name, value }));
 
-const components = {
+export const components = {
     Calendar: ~{
         type: 'datetime',
         format: ['m', '/', 'd', '/', 'yy'],
         filter: filter.Calendar },
     Char: ({ length }) => ({ filter: filter.Char(length) }),
     Checkbox: ~{ type: 'confirm' },
-    Color: ~{ type: 'chalk-pipe' },
+    Color: ~{
+        type: 'chalk-pipe',
+        validate: validator.Color },
```

Comment:
As long as I remember, this wasn't being used because `Color` can also be `red` or `violet`, for example. :thinking:

Reply:
Then we need to use something else than `isHexColor`, which was already defined (but unused, dead code) on `types.js`.